tests/alembic_migrations/versions/v_96729d6db8b3_convert_climo_obs_count_mv_to_native/test_smoke.py

```diff
@@ -0,0 +1,96 @@
+"""Smoke tests:
+- Upgrade drops table and creates matview
+- Downgrade drops matview and creates table
+"""
+
+# -*- coding: utf-8 -*-
+import logging
+import pytest
+from alembic import command
+
+import pycds.database
+from pycds.database import get_schema_item_names
+
+
+logger = logging.getLogger("tests")
+# logging.getLogger("sqlalchemy.engine").setLevel(logging.INFO)
+
+
+matviews = {"climo_obs_count_mv": {"indexes": {"climo_obs_count_idx"}}}
+matview_names = set(matviews)
+
+
+def check_matviews(engine, schema_name, present):
```

Comment:
Factor out as a fixture. This entire method is repeated here and in `tests/alembic_migrations/versions/v_7a3b247c577b_add_varsperhistory_native_matview/test_smoke.py`